### ISSUE TITLE
Option to include line numbering

### ISSUE THREAD

**Issue by rj93** [romannurik/SlidesCodeHighlighter#48]
It would be a nice to have the option to enable line numbering in the output

**Comment by romannurik** [comment]
I started on this in the `prism` branch but don't know if/when it'll be exposed.

**Comment by alan-barzilay** [comment]
has any progress been made in this direction?

**Comment by romannurik** [comment]
No, it's currently a hidden feature that needs a bit more work.

**Comment by abourget** [comment]
Oh how lovely it would be.

Awesome project btw, I love it.
